examples/remote_data_manager.py

```diff
@@ -0,0 +1,56 @@
+"""
+========================================
+Using sunpy.data.manager
+========================================
+
+This is an example to show how to use remote data manager to
+handle remote data in sunpy.
+"""
+# Remote data manager is used to handle the usage of remote files in the code
+# base in a sane way.
+##############################################################################
+from sunpy.data import manager
+
+##############################################################################
+# Let's start by defining a function which uses some remote files.
+# Let's assume the function requires a file, say http://212.183.159.230/5MB.zip, which has
+# the SHA1 hash '0cc897be1958c0f44371a8ff3dddbc092ff530d0'.
+
+@manager.require('test_file',
+                 ['http://212.183.159.230/5MB.zip'],
```

Comment:
What is this file?

Reply:
A test file that vodafone hosts.

Reply:
Could we use something else? Maybe one of the sample data files?